doc/deps.md

```diff
@@ -16,6 +16,8 @@ be installed from the official repository of the given Linux distribution.
 - **`libboost-all-dev`**: Boost can be used during the development.
 - **`llvm-7-dev`**, **`libclang-7-dev`**: C++ parser uses LLVM/Clang for
   parsing the source code.
+-**`clang-7`** this is only necessary during the install if you have another
```

Comment:
`llvm-7` and `clang-7` dependencies was removed recently, by @bruntib  if I remember correctly.
So do we still need them?